src/dataflow/metrics/mod.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright Materialize, Inc. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use lazy_static::lazy_static;
+
+use prometheus::{register_int_counter_vec, IntCounterVec};
+use prometheus_static_metric::make_static_metric;
+
+make_static_metric! {
+    pub struct EventsRead: IntCounter {
+        "format" => { avro, bytes, csv, protobuf },
```

Comment:
I _think_ that calling this `unknown` or `raw` so that it doesn't look like it is actually counting the number of bytes.

Reply:
This is supposed to reflect the `FORMAT ...` bit of the `CREATE SOURCE` statement, right? If so I think we should use "bytes" to keep it consistent. We've been very consistently avoiding "raw" in docs and SQL syntax.